lib/kitchen/driver/openstack.rb

```diff
@@ -82,23 +81,22 @@ def create(state)
             config[:server_name] = default_name
           end
         end
-        config[:disable_ssl_validation] && disable_ssl_validation
+        disable_ssl_validation if config[:disable_ssl_validation]
```

Comment:
To @smurawski's point, while TK doesn't complain if you pass in configuration options that are not specified, it's good practice to define a default so the code self-documents the fact that options are available.

I'd recommend putting this at the top of your class:

```ruby
default_config :disable_ssl_validation, false
```

That way it's 100% clear what the intention is, and you never have to worry about an edge case where this being nil bites you :)